src/utils.js

```diff
@@ -661,6 +661,23 @@ go.utils = {
             });
     },
 
+    post_nursereg: function(contact, im) {
+        var payload = {
```

Comment:
Thoughts on whether this should be ready to accept the additional fields if present? Or saving for later?